kubernetesevents/generic_handler.go

```diff
@@ -122,16 +122,22 @@ func (h *GenericHandler) add(selectorMap map[string]interface{}, metadata *model
 	}
 	serviceEvent.Service = service
 
-	namespace, err := h.kClient.Namespace.ByName(metadata.Namespace)
-	if err != nil {
-		return err
-	}
 	env := make(map[string]string)
-	env["name"] = namespace.Metadata.Name
-	env["externalId"] = "kubernetes://" + namespace.Metadata.Uid
+
+	if metadata.Namespace == "kube-system" {
```

Comment:
Here's I'm just hard coding if the namespace is kube-system, then just use that as the name and ID and don't look it up because the k8s API returns 404.